doc/manual/src/design/store/store.md

```diff
@@ -6,22 +6,21 @@ Every store also has a "store directory path", which is a path prefix used for v
 There are many types of stores, but all of them at least respect this model.
 Some however offer additional functionality.
 
-## A Rosetta stone for the Nix store.
+## A [Rosetta stone](https://en.m.wikipedia.org/wiki/Rosetta_Stone) for build system terminology
 
-The design of Nix is comparable to other build systems, even programming languages in general.
-Here is a rough [Rosetta stone](https://en.m.wikipedia.org/wiki/Rosetta_Stone) for build system terminology.
-If you are familiar with one of these columns, this might help the following sections make more sense.
+The Nix store's design is comparable to other build systems.
+Usage of terms is, for historic reasons, not entirely consistent within the Nix ecosystem, and still subject to slow change.
 
-generic build system | Nix | Bazel | Build Systems à la Carte | lazy programming language
--- | -- | -- | -- | --
-data (build input, build result) | component | file (source, target) | value | value
-build plan | derivation graph | action graph | `Tasks` | thunk
-build step | derivation | rule | `Task` | thunk
```

Comment:
I don't really like getting rid of this line. `derivation` is IMO more important than builder, and `thunk` is more important than `function`.

Reply:
Not sure what you mean, the `derivation` line is now second.

Reply:
I mean "build step" vs "build instruction".

Reply:
Sorry, I still don't understand. Which order do you want to see?